extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueLightHandler.java

```diff
@@ -93,13 +106,28 @@ public void initialize() {
                 updateStatus(statusInfo.getStatus(), statusInfo.getStatusDetail(), statusInfo.getDescription());
                 FullLight fullLight = getLight();
                 if (fullLight != null) {
+                    String modelId = fullLight.getModelID().replaceAll(NORMALIZE_ID_REGEX, "_");
+                    updateProperty(Thing.PROPERTY_MODEL_ID, modelId);
```

Comment:
Did you test this with things defined in a things file?
I actually think that https://github.com/eclipse/smarthome/issues/588 first needs to be implemented in order to make this work as expected.